workspace/data-sources/data.navigation.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+require_once(EXTENSIONS.'/page_lhandles/lib/class.datasource.MultilingualNavigation.php');
```

Comment:
This will crash if the extension is not present. We should keep the old commented part.